pages/team/index.tsx

```diff
@@ -1,11 +1,13 @@
 import { DOMAINS } from '@lib/data/domains'
+import { sortedTeamMembers } from 'utils/hierarchy'
 import { getTeamMembers } from '@lib/sanity-api'
 import Footer from '@shared/components/footer'
 import NewCarousel from '@shared/components/team/carousel'
 import MemberCard from '@shared/components/team/membercard'
 import Head from 'next/head'
 
 const Team = ({ teamMembers }) => {
+  teamMembers = sortedTeamMembers(teamMembers)
```

Comment:
instead of mutating the prop variable, for better clarity maybe storing the sorted result in a new variable?
cc: @sm-sami 

```suggestion
  const sortedTeamMembers = sortedTeamMembers(teamMembers)
```

Reply:
Ye, fewer the mutations, the better understandability. Get the output stored in a different variable and map over it